app/services/evacancy/ev_connect_service.rb

```diff
@@ -0,0 +1,64 @@
+module Evacancy
+  class EvConnectService
+    class << self
+      def check_ports
+        token = check_access_token
+
+        previously_open = 0
+        currently_open = 0
+
+        EvConnectStationPort.all.each do |station|
+          new_status = get_status(token, station.qr_code)
+          previously_open += 1 if station.port_status == 'AVAILABLE'
+          currently_open += 1 if new_status == 'AVAILABLE'
+          station.update(port_status: new_status)
+        end
+
+        if previously_open === 0 and currently_open > 0
```

Comment:
Since we keep track of each port individually, I'm wondering if we could also compare each one rather than comparing `previously_open` and `currently_open`. For example, I think we could notify again if we go from
* open, closed, closed, closed -> open, open, closed, closed
or maybe even
* open, closed, closed, closed -> closed, open, closed, closed
What do you think?

This adds more questions, so I'm also fine with pushing it to a future iteration

Reply:
One reason I didn't want to check for every change individually is that it might spam the channel too much. Since we have a queue now it might make sense to check for 2nd, 3rd, and 4th in line. Maybe if the first person in line does a /queue charging AND there's still a spot open it can notify the 2nd in line. We can add that in a future iteration.